pkgs/build-support/build-fhs-userenv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ writeTextFile, stdenv, ruby } : { env, runScript } :
+
+let
+  name = env.pname;
+
+  # Sandboxing script
+  chroot-user = writeTextFile {
+    name = "chroot-user";
+    executable = true;
+    destination = "/bin/chroot-user";
+    text = ''
+      #! ${ruby}/bin/ruby
+      ${builtins.readFile ./chroot-user.rb}
```

Comment:
Hm, I thought it would, too, but this file lies in `nixpkgs` and I supposed that we rely on it being unchanged while evaluating (otherwise lazy `import`s would be bad, too). Or have I completely misunderstood the issue?